torch/tensor.py

```diff
@@ -426,6 +426,10 @@ def __array_wrap__(self, array):
             array = array.astype('uint8')
         return torch.from_numpy(array)
 
+    def __contains__(self, val):
+        r"""Check if `val` is present"""
+        return (val == self).any().item()
```

Comment:
shouldn't you make sure that `val` is a pytorch/python(or numpy?) scalar first, and return NotImplemented otherwise?

Reply:
What about the case where we need to check if a row existed in a matrix?

Reply:
@SsnL  throwing NotImplemented in not tensor or scalar.
@vishwakftw NumPy also does not check if a row exists in a matrix.
Current behavior: any one element being present returns true.
Certainly, checking if row exists will be good to have. 
Question is whether to make it default or under some option.